docs/source/GETTING_STARTED.rst

```diff
@@ -115,7 +115,7 @@ You are now ready to start the Kiosk!
 DeepCell Kiosk Usage
 --------------------
 
-* Once the Kiosk has started, select the configuration option for your chosen cloud provider (currently, only Google Cloud is supported). The next screen will prompt you to authenticate your account with gcloud or to continue with a previously authenticated account. The next several screens will prompt you to select a gcloud project, name your cluster, and enter a bucket name for data storage. If you followed the Google Cloud Setup instructions from above, you should use that project and bucket name.
+* Once the Kiosk has started, select the Configure option for your chosen cloud provider (currently, only Google Kubernetes Engine is supported). The next screen will prompt you to authenticate your account with gcloud or to continue with a previously authenticated account. The next several screens will prompt you to select a gcloud project, name your cluster, and enter a bucket name for data storage. If you followed the Google Cloud Setup instructions from above, you should use that project and bucket name.
```

Comment:
Should this start: "Once the Kiosk Console has started..."

And I think our style dictates we use ... the `Configure`option... (all menu options have that formatting)